lib/member_page.rb

```diff
@@ -18,23 +18,23 @@ class MemberPage < Scraped::HTML
   end
 
   field :constituency do
-    box.css('center').text[/MP\ for(.*)/].tidy.chomp('.')
+    box.css('center').text[/(?<= MP\ for)(.*)/].tidy.chomp('.')
```

Comment:
This is quite over-complex. This would be much simpler as `text[/MP for (.*)/, 1]`